tests/test_e2e_12_mef_eline.py

```diff
@@ -0,0 +1,244 @@
+import unittest
```

Comment:
Hi Rogerio,
We are trying to have everything running following the Pytest framework concept, so do you are almost doing in this implementation.

Reply:
Hi @ArturoQuintana. 
All done.